### DISCUSSION TITLE
对话本背景

### DISCUSSION THREAD

**Discussion by ghost** [WatsonYao/nian_app#22] [version]
请问可以自定义对话本的背景吗？
或者是否可以直接应用“进展编辑页 - 自定义背景”？

**Comment by WatsonYao** [comment]
对话本支持自定义背景的，左右滑动看下 有记本编辑选项的

**Comment by ghost** [comment]
多谢大佬回复！
对话本现有的背景添加之后，只显示在头部，更像是记本的 Banner。
我说的背景是指标准记本的日间、夜间背景那种，在文本/对话输入界面的背景。

**Reply by WatsonYao** [reply]
对话页面的话，只可能考虑设定背景颜色，图片的话比较麻烦，可以接受嘛
